apps/pm-user/src/components/molecules/loginForm/LoginForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import styles from './loginForm.module.scss';
+import { Button, Input } from '../..';
+
+const LoginForm = ({ onLogin }) => {
+  const [formValues, setFormValues] = useState({
+    emailValue: '',
+    passwordValue: '',
+  });
+
+  function submitForm(e) {
+    e.preventDefault();
+
+    if (formValues) {
+      onLogin(formValues.passwordValue, formValues.emailValue);
+    } else {
+      console.log('email or password can not be empty');
+    }
+  }
+
+  function getFormValues(e) {
+    const value = e.target.value;
+    const name = e.target.name;
+
+    setFormValues((oldVlaues) => {
+      return { ...oldVlaues, [name]: value };
+    });
+  }
+
+  return (
+    <div className={styles.super_container}>
```

Comment:
this code should not be in the login form

Reply:
I didn't write this code originally Vinit. Hence please look into this issue.

Reply:
Resolved issue. No need of async here.